Typo in a candidat project name for the Quality Award 2020

There is a typo in the name of a project listed for the Software Quality Award 2020 [here](https://www.yegor256.com/2019/11/03/award-2020.html)

`altiore/altiore.ui` should be `altiore/lorder.ui`

There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2cbcf229-0ac2-46cd-a04b-76a0f998dfc0, thread: pool-5-thread-1
org.hamcrest.MatcherAssert[20] java.lang.AssertionError: HTTP response status is not equal to 201:
404 Not Found [https://api.github.com/repos/yegor256/blog/labels]
Server: GitHub.com
Date: Fri, 02 Oct 2020 18:47:20 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-GitHub-Media-Type: github.v3
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4479
X-RateLimit-Reset: 1601664497
X-RateLimit-Used: 521
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: A664:137F:D6C2A7:16BCA05:5F7775B8

{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/issues#create-a-label"}
Expected: HTTP response with status 201
     but: was <404 Not Found [https://api.github.com/repos/yegor256/blog/labels]
Server: GitHub.com
Date: Fri, 02 Oct 2020 18:47:20 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-GitHub-Media-Type: github.v3
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4479
X-RateLimit-Reset: 1601664497
X-RateLimit-Used: 521
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: A664:137F:D6C2A7:16BCA05:5F7775B8

{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/issues#create-a-label"}>
```

[0.54.5](https://github.com/zerocracy/farm): Issue: yegor256/blog#448, Action: `opened`

--- a/_posts/2019/nov/2019-11-03-award-2020.md
+++ b/_posts/2019/nov/2019-11-03-award-2020.md
@@ -96,7 +96,7 @@ By the way, if you want to sponsor this award and increase the bonus,
 There were 67 projects submitted.
 33 of them participated in the competition (in alphabetic order):
 
-  * [`altiore/altiore.ui`](https://github.com/altiore/altiore.ui) — @fellahi-ali
+  * [`altiore/lorder.ui`](https://github.com/altiore/lorder.ui) — @fellahi-ali
   * [`btraceio/btrace`](https://github.com/btraceio/btrace) - @iakunin
   * [`catalyst-team/catalyst`](https://github.com/catalyst-team/catalyst) — @iakunin
   * [`coderaiser/cloudcmd`](https://github.com/coderaiser/cloudcmd) — @razzwan

